README.md

```diff
@@ -24,8 +25,36 @@ pen & paper RPG game design".
 
 ## Setup
 To install required packages, run
+```shell
+pip install -r requirements.txt
+```
 
-    pip install -r requirements.txt
+
+## Usage
+### Server
+To start server, first enter `serving` directory:
+```shell
+cd serving
+```
+
+Then run following command:
+```shell
+uvicorn server:app --reload
+```
+The frontend app should now be available on [localhost:8000](http://localhost:8000).
+
+### Frontend
+Frontend is written in React.js. To run it, you will need `npm` downloaded from [nodejs.org](https://nodejs.org/en/download).
+
+Then, to install project's dependencies and start server, run:
+```shell
+cd serving/frontend
+npm install
+npm start
+```
+After `npm` finishes setup, the frontend view will be available on [localhost:3000](http://localhost:3000/).
+
```

Comment:
Wy nie macie tego w Dockerze? Relatywnie prosto się to stawia, a szczególnie do JSa to polecam, bo NPM jest... przeciętnie bezpieczny

Reply:
Można to też spiąć z Docker Compose, żeby od razu wstawały backend i frontend

Reply:
A w ogóle taki "master setup" to jeszcze ma Makefile, żeby np. zrobić `make frontend`, `make backend`, `make start`, `make test` i inne wygodne rzeczy